src/xml.tsx

```diff
@@ -485,6 +436,16 @@ export function parse(source: string, middleware?: Middleware): JsxAST | null {
     return name;
   }
 
+  function getTag(
+    name: keyof typeof tags
+  ): ComponentType<{ children?: React.ReactNode }> {
```

Comment:
I don't think that `ComponentType<{ children?: React.ReactNode }>` is the right type for Svg tags, i.e. `Circle` or `Rect`

Reply:
removed that function.